src/Fhir.Anonymizer.Tool/FilesAnonymizerForJsonFormatResource.cs

```diff
@@ -124,5 +122,23 @@ private string GetResourceOutputFileName(string fileName, string inputFolder, st
 
             return Path.Combine(outputFolder, partialFilename);
         }
+
+        private AnonymizerEngine CreateAnonymizerEngineForFile(string filePath)
```

Comment:
Shall we add a static CreateForResourceFile(stirng configPath, string resourceFileName) function in AnonymizerEngine to reuse this logic?

Reply:
Agree! It is added in the update.